crates/router/src/core/payouts.rs

```diff
@@ -1298,7 +1298,7 @@ pub async fn complete_create_payout(
             let db = &*state.store;
             let payout_attempt = &payout_data.payout_attempt;
             let updated_payout_attempt = storage::PayoutAttemptUpdate::StatusUpdate {
-                connector_payout_id: "".to_string(),
+                connector_payout_id: None,
```

Comment:
can we map it to `payout_data.payout_attempt.connector_payout_id` instead to hardcoding to None